src/c++/perf_analyzer/client_backend/client_backend.cc

```diff
@@ -45,13 +45,9 @@ namespace triton { namespace perfanalyzer { namespace clientbackend {
 //================================================
 
 const Error Error::Success("", pa::SUCCESS);
-const Error Error::Failure("", pa::DEFAULT_ERROR);
+const Error Error::Failure("", pa::GENERIC_ERROR);
 
-Error::Error()
-{
-  msg_ = "";
-  error_ = 0;
-}
+Error::Error() : msg_(""), error_(0) {}
```

Comment:
Could the `0` here be changed to `pa::SUCCESS`?

Reply:
done